src/full/Agda/TypeChecking/Coverage.hs

```diff
@@ -383,19 +378,12 @@ cover f cs sc@(SClause tel ps _ _ target) = updateRelevance $ do
 
         reportSDoc "tc.cover.applyCl" 40 $ "new ty =" <+> pretty ty
 
-        return $
-             Clause { clauseLHSRange  = clauseLHSRange cl
-                    , clauseFullRange = clauseFullRange cl
-                    , clauseTel       = tel
-                    , namedClausePats = (s `applySubst` ps) ++ extra
-                    , clauseBody      = (`applyE` patternsToElims extra) . (s `applyPatSubst`) <$> clauseBody cl
-                    , clauseType      = ty
-                    , clauseCatchall  = clauseCatchall cl
-                    , clauseExact     = clauseExact cl
-                    , clauseRecursive = clauseRecursive cl
-                    , clauseUnreachable = clauseUnreachable cl
-                    , clauseEllipsis  = clauseEllipsis cl
-                    }
+        return $ cl
```

Comment:
Spelling out all the fields is our preferred style.  This way, the compiler gives a warning when a new field is added.  Please revert.  (Here and in the other instances.)